htlcswitch/iterator.go

```diff
@@ -137,6 +158,18 @@ func (r *sphinxHopIterator) ForwardingInstructions() ForwardingInfo {
 	}
 }
 
+// ExtractErrorEncrypter decodes and returns the ErrorEncrypter for this hop,
+// along with a failure code to signal if the decoding was successful. The
+// ErrorEncrypter is used to encrypt errors back to the sender in the event that
+// a payment fails.
+//
+// NOTE: Part of the HopIterator interface.
+func (r *sphinxHopIterator) ExtractErrorEncrypter(
+	extracter ErrorEncrypterExtracter) (ErrorEncrypter, lnwire.FailCode) {
+
```

Comment:
nit: extra newline

Reply:
was under the impression that if the function signature wraps, we typically insert an extra new line to separate the code from the arguments, though either works for me

Reply:
I have no idea :p